namespace cpp dsn

// place holder
struct rpc_address
{
}

// place holder
struct blob
{
}

// place holder
struct error_code
{
}

